### PR TITLE
UID2-532 Update instructions for local docker testing

### PR DESCRIPTION
**Pull request by cYKatherine**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,5 +40,11 @@ mvn clean compile exec:java -Dvertx-config-path=conf/local-config.json
 ```
 mvn clean compile exec:java -Dvertx-config-path=conf/integ-config.json
 ```
+## Local deployment/testing on Docker
+1. Change `COPY ./conf/default-config.json /app/conf/` in line 13 of `Dockerfile` to `COPY ./conf/local-config.json /app/conf/`
+2. Run ```mvn package```
+3. Run ```docker build -t uid-operator --build-arg JAR_VERSION=1.11.0 .```
```
Comment (thomasm-ttd) [inline README.md:46]: I would change this to detail how to find the current version. The version changes regularly. It is set in the pom.xml file.

File: README.md [hunk 2]
```diff
@@ -40,5 +40,11 @@ mvn clean compile exec:java -Dvertx-config-path=conf/local-config.json
 ```
 mvn clean compile exec:java -Dvertx-config-path=conf/integ-config.json
 ```
+## Local deployment/testing on Docker
+1. Change `COPY ./conf/default-config.json /app/conf/` in line 13 of `Dockerfile` to `COPY ./conf/local-config.json /app/conf/`
```
Comment (thomasm-ttd) [inline README.md:44]: Change the end of this to COPY ./conf/docker-config.json /app/conf/default-config.json

**Review by thomasm-ttd — COMMENTED** [review]
Don't make changes to the local-config.json file. Rather create a new file called docker-config.json, and in your instructions in the Readme, use that file.

File: README.md (hunk 2 above)
Comment (cYKatherine) [reply README.md]: Done!

File: README.md (hunk 1 above)
Comment (cYKatherine) [reply README.md]: Done!

**Review by thomasm-ttd — APPROVED** [review]

**Pull request merged**